service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/HearingBookingDetailsControllerTest.java

```diff
@@ -31,7 +38,7 @@ void shouldReturnAnErrorWhenHearingDateIsSetToYesterday() throws Exception {
         LocalDateTime yesterday = LocalDateTime.now().minusDays(1);
 
         AboutToStartOrSubmitCallbackResponse callbackResponse = makeRequest(createHearing(
-            yesterday, yesterday.plusDays(1)));
+            yesterday, yesterday.plusDays(1)), EMPTY_BOOKING);
```

Comment:
inconsistent naming, first param execute method with name about hearing, second param is about booking, but essentially these are same type of things, maybe EMPTY_HEARING_BOOKING.